application/cms/forms.py

```diff
@@ -118,7 +118,10 @@ class DataSourceForm(FlaskForm):
     frequency_of_release_id = RDURadioField(
         label="How often is the source data published?",
         coerce=int,
-        validators=[InputRequired("Select the source data publication frequency")],
+        validators=[
+            InputRequired("Select the source data publication frequency"),
+            FrequencyOfReleaseOtherRequiredValidator(),
```

Comment:
I think the order of validators can be important - maybe double check if you need to move it, but I think it was the first validator before you removed it.